elliott/elliottlib/cli/find_builds_cli.py

```diff
@@ -224,22 +230,82 @@ async def find_builds_cli(runtime: Runtime, advisory, default_advisory_type, bui
         else:  # attach
             erratum.attach_builds(unshipped_builds, kind)
             cdn_repos = et_data.get('cdn_repos')
-            if cdn_repos and not no_cdn_repos and kind == "image":
-                erratum.set_cdn_repos(cdn_repos)
+            if kind == 'image':
+                set_rhcos_file_meta(advisory_id)
+                if cdn_repos and not no_cdn_repos:
+                    erratum.set_cdn_repos(cdn_repos)
+
     except ErrataException as e:
-        red_print(f'Cannot change advisory {advisory}: {e}')
+        red_print(f'Cannot change advisory {advisory_id}: {e}')
         exit(1)
 
 
-def _fetch_nvrps_by_nvr_or_id(ids_or_nvrs, tag_pv_map, ignore_product_version=False, brew_session: koji.ClientSession = None):
-    session = koji.ClientSession(constants.BREW_HUB)
-    builds = brew.get_build_objects(ids_or_nvrs, session)
+def get_rhcos_nvrs_from_assembly(runtime: Runtime, brew_session: koji.ClientSession = None):
+    rhcos_config = assembly_rhcos_config(runtime.get_releases_config(), runtime.assembly)
+    build_ids_by_arch = dict()
+    nvrs = []
+    for _, tag_config in rhcos_config.items():
+        for arch, pullspec in tag_config['images'].items():
+            build_id = get_build_id_from_rhcos_pullspec(pullspec, runtime.logger)
+            if arch not in build_ids_by_arch:
+                build_ids_by_arch[arch] = set()
+            build_ids_by_arch[arch].add(build_id)
+
+    for arch, builds in build_ids_by_arch.items():
+        for build_id in builds:
+            nvr = f'rhcos-{arch}-{build_id}'
+            if brew_session.getBuild(nvr):
+                runtime.logger.info(f'Found rhcos nvr: {nvr}')
+                nvrs.append(nvr)
+            else:
+                runtime.logger.info(f'rhcos nvr not found: {nvr}')
```

Comment:
Since we ignore the nvr not found error, it should be better if we make this a warning.

Reply:
Updated